implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -209,16 +209,23 @@ impl<'a> NodeManagerProjectsOptions<'a> {
     }
 }
 
+#[derive(Clone)]
+pub struct ApiTransport {
+    /// Type of transport being requested
+    pub tt: TransportType,
+    /// Mode of transport being requested
+    pub tm: TransportMode,
+    pub socket_address: Address,
+    pub worker_address: String,
```

Comment:
Docs for the fields are not finished due to the naming discrepancies [mentioned below](https://github.com/build-trust/ockam/pull/4452/files#r1133221415). 

Also, how detailed do we want to make the docs for the fields?

Reply:
Not too much, since the types are pretty common.